lib/modules/apostrophe-notifications/views/notification.html

```diff
@@ -1,3 +1,3 @@
 <div class="apos-ui apos-notification apos-notification--hidden" data-apos-notification>
-  <p class="apos-notification-message" data-apos-notification-message>{{ data.message }}</p>
+  <p class="apos-notification-message" data-apos-notification-message>{{ data.message }}<span class="apos-notification-message-close"><i class="fa fa-times-circle" aria-hidden="true"></i></span></p>
```

Comment:
We should use a `button` here for semantics and the built-in benefits. Or maybe the `div` around the whole thing should be a `button` if clicking the whole thing closes it. That'd only be bad if auto-dismiss ones don't dismiss on click too.

Reply:
Holler if you want me to take a swipe at that.

Reply:
They all dismiss on click. Feel free to make it a button and get the styles happy with that. Thanks.

Reply:
Agree we should do what's expedient in this PR (fill the current UX hole) and continue to iterate. Button seems reasonable to do now.